contracts/SSVNetwork.sol

```diff
@@ -82,7 +83,8 @@ contract SSVNetwork is UUPSUpgradeable, Ownable2StepUpgradeable, ISSVNetwork {
         uint64 operatorMaxFeeIncrease_,
         uint64 declareOperatorFeePeriod_,
         uint64 executeOperatorFeePeriod_,
-        uint64 minimumBlocksBeforeLiquidation_
+        uint64 minimumBlocksBeforeLiquidation_,
+        uint64 minimumLiquidationCollateral_
```

Comment:
If we want to take minimumLiquidationCollateral as input in the initializer function, its mandatory to use a separate contract with a new initializer. So then we can have the changes in current stage deployed contracts.
But for the audit and mainnet deployment, we can go with this implementation.